package.json

```diff
@@ -29,6 +30,8 @@
     "@guardian/prettier": "^0.6.0",
     "eslint": "^7.31.0",
     "eslint-plugin-react": "^7.24.0",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "ts-loader": "^9.2.5",
+    "webpack": "^5.65.0"
```

Comment:
I had to set ts-loader/webpack versions here, otherwise it breaks the build of `server`. This is because storybook brings in webpack 4 and we end up with an incompatibility with ts-loader...